GameLiftPlugin/GameLiftPlugin.uplugin

```diff
@@ -58,4 +58,4 @@
 	"IsBetaVersion": false,
 	"IsExperimentalVersion": false,
 	"Installed": false
-}
\ No newline at end of file
+}
```

Comment:
Lets keep the new line at the end of the file here, not having this line at the end can cause issues in some Operating Systems